docs_src/components/Accordion.svelte

```diff
@@ -36,7 +36,9 @@
         <td>tabs</td>
         <td>ITabsConfig[]</td>
         <td />
-        <td>Array of tab data. See <Badge type="secondary" extension="small">Tab</Badge> props below for details</td>
+        <td
+          >Array of tab data. See <span class="badge badge--small" data-type="secondary">Tab</span> props below for details</td
+        >
```

Comment:
Hvorfor gør den det her? Det gør min editor også en gang i mellem. Bliver indholdet for langt eller hva?

Reply:
Ja, ser ud til der er et limit på spaces og så breaker den ikke særligt pænt

Reply:
hvis man selv laver breaks ved større-end og mindre-end, så breaker den pænt